roles/openshift_login/tasks/main.yml

```diff
@@ -98,22 +98,23 @@
     dest: "{{ cifmw_basedir|default(ansible_user_dir ~ '/ci-framework-data') }}/artifacts/parameters/openshift-login-params.yml"
     content: "{{ cifmw_openshift_login_params_content | from_yaml | to_nice_yaml }}"
 
-- name: Read the install yamls parameters file
-  ansible.builtin.slurp:
-    path: "{{ cifmw_basedir|default(ansible_user_dir ~ '/ci-framework-data') }}/artifacts/parameters/install-yamls-params.yml"
-  register: cifmw_openshift_login_install_yamls_artifacts_slurp
+- when: cifmw_install_yamls_environment is defined
+  block:
+  - name: Read the install yamls parameters file
```

Comment:
This isn't properly aligned - you can push the whole block content by one more indentation (2 spaces).

Reply:
In principle, you're onboard with this change, right?